### PR TITLE
Add 2019 roadmap

### PR DESCRIPTION
**Pull request by BenTheElder**
NOTE: this is not 100% exhaustive or complete, this is just a first pass.

Eventually we will eliminate todo.md in favor of the roadmap and github issues.

/kind documentation
cc @munnerz

### CODE REVIEW

**Comment by BenTheElder** [comment]
Also we need to get other issues listed here, but explicitly mentioned so far:
https://github.com/kubernetes-sigs/kind/issues/117
https://github.com/kubernetes-sigs/kind/issues/166
https://github.com/kubernetes-sigs/kind/issues/155
https://github.com/kubernetes-sigs/kind/issues/197
https://github.com/kubernetes-sigs/kind/issues/200

File: docs/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+and Phippy is awesome ❤️
+
+Longer Term goals include:
+- Enabling a suitable local storage provider for testing applications that need
+persistent storage
+- Improving "kind as a library"
+
+Misc:
+ - setup a regular Zoom meeting for the project
+ - acheive certified Kubernetes conformance
```
Comment (krzyzacy) [inline docs/roadmap.md]: achieve :-)

Comment (BenTheElder) [reply docs/roadmap.md]: thanks! fixed and ran misspell over it to check for anything else

**Comment by krzyzacy** [comment]
the goals sgtm, though I might not have all the context :-) punt to @munnerz and @neolit123

**Comment by munnerz** [comment]
This all looks good to me - do we want to clarify that 1.0 status = a versioned and supported API? (i.e. cutting a `kind.sigs.k8s.io/v1(beta1?)`

/lgtm
/hold

(holding for @neolit123 to have a chance to look!)

**Pull request merged**